src/Controller/Controller_Zapier.php

```diff
@@ -38,22 +39,35 @@ public function init(): void {
 	 *
 	 * @return array
 	 *
-	 * @throws \GFPDF_Vendor\Spatie\UrlSigner\Exceptions\InvalidExpiration
-	 * @throws \GFPDF_Vendor\Spatie\UrlSigner\Exceptions\InvalidSignatureKey
+	 * @throws \Exception
 	 * @since 6.1
 	 */
 	public function add_expanded_zapier_support( $body, $feed, $entry, $form ) {
 		$zapier = \GF_Zapier::get_instance();
 		$signer = new Helper_Url_Signer();
 		$model  = \GPDFAPI::get_mvc_class( 'Model_PDF' );
+		$logger = new Helper_Logger( 'gravity-pdf', 'Gravity PDF' );
```

Comment:
You need to pass `$log` from the bootstrap file to this class's `__construct()`.